documentation/recovery.md

```diff
@@ -14,36 +14,62 @@ The new system needs to be provisioned with the same certificate name as the sys
 This procedure uses the following placeholder references.
 
 * _\<primary-server-fqdn\>_ - The FQDN and certname of the primary Puppet server
-* _\<replacement-replica-fqdn\>_ - The FQDN and certname of the replacement replica Puppet server
-* _\<replacement-avail-group-letter\>_ - Either A or B; whichever of the two letter designations is appropriate for the server being replaced. It will be the opposite of the primary server.
+* _\<old-replica-fqdn\>_ - The FQDN and certname of the old replica Puppet server that has failed or is missing
+* _\<replacement-replica-fqdn\>_ - The FQDN and certname of the new replica Puppet server
+* _\<failed-primary-server-fqdn\>_ - The FQDN and certname of the original primary server that the old replica had replaced
+* _\<replacement-avail-group-letter\>_ - Either A or B; whichever of the two letter designations is appropriate for the replacement server. It will be the opposite of the server that it is replacing. 
 
-1. Ensure the old replica server is forgotten.
+1. If applicable, purge the failed primary server. (You may need to do this, for example, if the original primary failed and the promoted replica that replaced it has also failed.)
 
-        puppet infrastructure forget <replacement-replica-fqdn>
+        puppet node purge <failed-primary-server-fqdn>
 
-2. Install the Puppet agent on the replacement replica
+2. Ensure the old replica server is forgotten.
+
+        puppet infrastructure forget <old-replica-fqdn>
+
+3. Install the Puppet agent on the replacement replica.
+
+**Note**: When designating the availability group of the replacement, use the opposite group (A or B) of the server being replaced. This means that, if the old replica server replaced the original primary server, the new replica is assigned the same availability group as the original primary.
```

Comment:
Suggest deleting this line. It won't always be the opposite of the server being replaced but it should be opposite of whatever is the primary which is already stated in the former line 18. I suggest leaving that line there <replacement-avail-group-letter\>_ - Either A or B; whichever of the two letter designations is appropriate for the server being replaced. It will be the opposite of the primary server.